src/main.cpp

```diff
@@ -35,6 +39,12 @@ int main(void)
     }
 #endif
 
+#ifdef CONFIG_AOS_REBOOT
+    if (auto ret = start_reboot_monitoring()) {
+        printk("failed to start reboot monitoring (%d)\n", ret);
```

Comment:
Start with capital letter. Also fix for `create_domains`.

Reply:
Done